support-frontend/conf/riff-raff.yaml

```diff
@@ -7,7 +7,6 @@ deployments:
       bucket: support-frontend-static
       prefixPackage: false
       prefixStack: false
-      prefixStage: true
```

Comment:
this is riffraff's default and we get warnings if we confirm it in the deploy file 😢